components/calendar/src/types.rs

```diff
@@ -187,6 +188,37 @@ dt_unit!(
     "An ISO-8601 second component, for use with the [`IsoTimeInput`]."
 );
 
+#[derive(Debug)]
+pub struct Time {
```

Comment:
Question: why did you name the fields with `Iso` but not the struct?

Reply:
The fields were preexisitng in DTF, I don't think we plan to support notions of time other than hour-minute-second-(subsecond), so IsoTime seems redundant to me. It might be worth renaming the other fields to not have Iso but that sounds like a good-first-bug followup to file. I could also be convinced that this should be called IsoTime, but I like the symmetry of Date+Time=DateTime.

Reply:
Okay, it's fine.  Time is parallel with Date+Time, and the fields are represented as ISO.